### PR TITLE
[ES] Add `Legal/Terms`

### PR DESCRIPTION
**Pull request by agatemosu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Legal/Terms/es.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+Esta obligación de defensa e indemnización seguirá vigente tras la expiración de las presentes términos del servicio y tras el uso por su parte del Sitio web de osu!.
+
+## SU USO DE LOS CONTENIDOS DEL SITIO
+
+Usted comprende que al usar el sitio web de osu! estará expuesto a Envíos de Usuarios procedentes de diversas fuentes, y que osu! no es responsable de la precisión, utilidad, seguridad o derechos de propiedad intelectual de o relacionados con dichos Envíos de Usuarios.
+Además, entiende y reconoce que puede estar expuesto a Envíos de Usuario que sean imprecisos, ofensivos, indecentes o censurables, y acepta renunciar, y por la presente renuncia, a cualquier derecho o recurso legal o equitativo que tenga o pueda tener contra osu! con respecto a ello, y acepta indemnizar y absolver a osu!, sus Propietarios/Operadores, afiliados y/o licenciantes, en la máxima medida permitida por la ley con respecto a todos los asuntos relacionados con su uso del sitio.
+
+## ENVÍOS DE USUARIOS Y ELIMINACIÓN DE CONTENIDOS
```
Comment (UnknownOsu) [inline wiki/Legal/Terms/es.md:72]:
```suggestion
## ENVÍOS DE USUARIOS Y ELIMINACIÓN DE CONTENIDO
```

File: wiki/Legal/Terms/es.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+En relación con los Envíos de Usuario, usted acepta además que no enviará material protegido por derechos de autor, protegido por secreto comercial o sujeto de otro modo a derechos de propiedad de terceros, incluidos los derechos de privacidad y publicidad, a menos que usted sea el propietario de dichos derechos o tenga permiso de su legítimo propietario para publicar el material y conceder a osu! todos los derechos de la licencia otorgados en el presente documento.
+
+osu! no respalda ningún Envío de Usuario ni ninguna opinión, recomendación o consejo expresado en el mismo, y osu! declina expresamente toda responsabilidad en relación con los Envíos de Usuario.
+osu! no permite actividades que infrinjan los derechos de autor y los derechos de propiedad intelectual en su Sitio web, y osu! eliminará todos los Contenidos y Envíos de Usuario si se le notifica debidamente que dichos Contenidos o Envíos de Usuario infringen los derechos de propiedad intelectual de terceros.
```
Comment (UnknownOsu) [inline wiki/Legal/Terms/es.md:80]:
```suggestion
osu! no permite actividades que infrinjan los derechos de autor y los derechos de propiedad intelectual en su Sitio web, y osu! eliminará todo el Contenido y Envíos de Usuario si se le notifica debidamente que dicho Contenido o Envíos de Usuario infringen los derechos de propiedad intelectual de terceros.
```

File: wiki/Legal/Terms/es.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+osu! se reserva el derecho a eliminar Contenidos y Envíos de Usuario sin previo aviso.
```
Comment (UnknownOsu) [inline wiki/Legal/Terms/es.md:81]:
```suggestion
osu! se reserva el derecho a eliminar Contenido y Envíos de Usuario sin previo aviso.
```

File: wiki/Legal/Terms/es.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+- [El Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team)
+
+Los usuarios menores de 18 años no podrán ser miembros de ninguno de los equipos mencionados.
+
+## POLÍTICAS DE PRIVACIDAD
+
+osu! considera que su privacidad es lo más importante, y valoramos el uso que hace de nuestros servicios.
+Los servicios de osu! son gratuitos y no es necesario registrarse previamente para usarlos.
+osu! solicitará explícitamente información sobre usted cuando dicha información sea necesaria para el uso de los servicios prestados por osu!.
+El envío de cualquier tipo de información personal se realiza únicamente con su consentimiento (sitio web) o de forma automática (software cliente del juego) cuando es necesario para proporcionar diagnósticos y comentarios.
```
Comment (UnknownOsu) [inline wiki/Legal/Terms/es.md:97]:
```suggestion
El envío de cualquier tipo de información personal se realiza únicamente con su consentimiento (sitio web) o de forma automática (software del cliente del juego) cuando es necesario para proporcionar diagnósticos y comentarios.
```

File: wiki/Legal/Terms/es.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+osu! mantiene toda la información privada y personal en privado, y no comparte ni compartirá esta información con terceros, excepto en las siguientes circunstancias:
+
+Cuando dicha acción sea necesaria para cumplir con requisitos legales o procesos judiciales.
+
+Cuando dicha acción sea necesaria para proteger y defender los derechos o la propiedad de osu!.
+
+Cuando dicha acción sea necesaria para hacer cumplir los términos y condiciones del servicio de osu!.
+
+osu! puede usar cookies como parte del proceso de seguimiento, pero las cookies no se usarán para identificarle personalmente sin su consentimiento previo.
```
Comment (UnknownOsu) [inline wiki/Legal/Terms/es.md:106]:
```suggestion
osu! puede usar cookies como parte del proceso de seguimiento, pero las cookies no se usarán para identificarlo personalmente sin su consentimiento previo.
```

File: wiki/Legal/Terms/es.md [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+Si el usuario decide participar en el foro, con su consentimiento, se emitirá una cookie para realizar un seguimiento del nombre de usuario y la contraseña durante el periodo de 1 año.
+
+## DERECHOS DE PROPIEDAD DE LOS CONTENIDOS
```
Comment (UnknownOsu) [inline wiki/Legal/Terms/es.md:109]:
```suggestion
## DERECHOS DE PROPIEDAD DEL CONTENIDO
```

**Review by UnknownOsu — CHANGES_REQUESTED** [review]

File: wiki/Legal/Terms/es.md [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
...S, EMPLEADOS O AGENTES, SERÁN RESPONSABLES ANTE USTED POR NINGÚN DAÑO DIRECTO, INDIRECTO, INCIDENTAL, ESPECIAL, PUNITIVO O CONSECUENTE QUE RESULTE DE (I) ERRORES, EQUIVOCACIONES O INEXACTITUDES DE CONTENIDO, (II) LESIONES PERSONALES O DAÑOS A LA PROPIEDAD, DE CUALQUIER NATURALEZA, QUE RESULTEN DE SU ACCESO Y USO DE NUESTRO sitio web, (III) CUALQUIER ACCESO NO AUTORIZADO O USO DE NUESTROS SERVIDORES SEGUROS Y/O CUALQUIER Y TODA LA INFORMACIÓN PERSONAL Y/O INFORMACIÓN FINANCIERA ALMACENADA EN EL MISMO, (IV) CUALQUIER INTERRUPCIÓN O CESE DE LA TRANSMISIÓN HACIA O DESDE NUESTRO sitio web, (V) CUALQUIER ERROR, VIRUS, TROYANOS O SIMILARES, QUE PUEDAN SER TRANSMITIDOS HACIA O A TRAVÉS DE NUESTRO sitio web POR CUALQUIER TERCERO, Y/O (VI) CUALQUIER ERROR U OMISIÓN EN CUALQUIER CONTENIDO O POR CUALQUIER PÉRDIDA O DAÑO DE CUALQUIER TIPO INCURRIDOS COMO RESULTADO DEL USO QUE USTED HAGA DE CUALQUIER CONTENIDO PUBLICADO, ENVIADO POR CORREO ELECTRÓNICO, TRANSMITIDO O PUESTO A DISPOSICIÓN DE OTRA MANERA A TRAVÉS DEL sitio web osu!, YA SEA POR GARANTÍA, CONTRATO, AGRAVIO O CUALQUIER OTRA TEORÍA LEGAL, E INDEPENDIENTEMENTE DE QUE LA EMPRESA HAYA SIDO ADVERTIDA O NO DE LA POSIBILIDAD DE DICHOS DAÑOS.
```
[the remark this answers is not in the document]
Comment (agatemosu) [reply wiki/Legal/Terms/es.md:52]: Hice un reemplazar todo y se perdieron las mayusculas xd

File: wiki/Legal/Terms/es.md [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+
+## DERECHOS DE PROPIEDAD DEL CONTENIDO
+
+El Usuario reconoce que el contenido, incluyendo, pero no limitado a texto, software, música, sonido, fotografías, vídeo, gráficos, referencias de nombres u otro material contenido en los resultados de búsqueda de anuncios de patrocinadores, proporcionado al Usuario o al que este acceda a través del Servicio («Contenido») por osu!, está protegido por derechos de autor, marcas registradas, marcas de servicio, patentes u otros derechos y leyes de propiedad; por lo tanto, el Usuario solo está autorizado a usar este Contenido según lo expresamente autorizado por el Servicio.
+El Usuario no podrá copiar, reproducir, distribuir o crear trabajos derivados de este Contenido sin estar expresamente autorizado para ello por el Servicio.
+
+## ENLACES
+
+osu! puede proporcionar, o los Usuarios pueden incluir en el correo electrónico, enlaces a otros sitios web o recursos.
+El usuario reconoce y acepta que osu! no es responsable de la disponibilidad de dichos sitios o recursos externos, y no respalda ni es responsable de ningún contenido, publicidad, productos u otros materiales en dichos sitios o recursos o disponibles a través de ellos.
```
Comment (UnknownOsu) [inline wiki/Legal/Terms/es.md:117]:
```suggestion
El Usuario reconoce y acepta que osu! no es responsable de la disponibilidad de dichos sitios o recursos externos, y no respalda ni es responsable de ningún contenido, publicidad, productos u otros materiales en dichos sitios o recursos o disponibles a través de ellos.
```

File: wiki/Legal/Terms/es.md [hunk 9]
```diff
@@ -0,0 +1,118 @@
...
+El usuario acepta que osu! no será responsable, directa o indirectamente, de ningún daño o pérdida causados o supuestamente causados por o en conexión con el uso o la confianza depositada en cualquier contenido, bien o servicio disponible en dichos sitios o recursos externos.
```
Comment (UnknownOsu) [inline wiki/Legal/Terms/es.md:118]:
```suggestion
El Usuario acepta que osu! no será responsable, directa o indirectamente, de ningún daño o pérdida causados o supuestamente causados por o en conexión con el uso o la confianza depositada en cualquier contenido, bien o servicio disponible en dichos sitios o recursos externos.
```

**Review by UnknownOsu — CHANGES_REQUESTED** [review]

File: wiki/Legal/Terms/es.md (hunk 5 above)
Comment (UnknownOsu) [reply wiki/Legal/Terms/es.md:106]: @agatemosu También este

**Review by UnknownOsu — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**